2Homework25.02.22/Trie/Trie/Trie.cs

```diff
@@ -0,0 +1,148 @@
+namespace Trie;
+
+/// <summary>
+/// data structure for storing a set of strings
+/// </summary>
+public class Trie
+    {
+        private class Node
+        {
+            public Node()
+            {
+                this.Next = new Dictionary<char, Node>();
+            }
+            public short NumberOfWords;
+            public bool IsTerminal;
+            public Dictionary<char, Node> Next;
+        }
+        public Trie()
+        {
+            Head = new Node();
+        }
+
+        private Node Head;
```

Comment:
```suggestion
        private Node head;
```
Поля (кроме public-ов) именуются со строчной всегда